update.sh

```diff
@@ -17,7 +17,7 @@ echo "Updating certificates..."
 ./certs/update.sh
 
 echo "Fetching and building traefik $VERSION..."
-wget -O traefik https://github.com/emilevauge/traefik/releases/download/$VERSION/traefik
+wget -O traefik https://github.com/containous/traefik/releases/download/$VERSION/traefik
```

Comment:
@timoreimann I'm not extraordinarily familiar with wget, but I don't see `--fail` as an option [in their manual](https://www.gnu.org/software/wget/manual/wget.html). In addition, I did some testing on this and wget will return a non-zero error code when the file is not found, meaning `set -e` is effective when the file is not found. What type of failure are you hoping to prevent?